api/build.rs

```diff
@@ -15,8 +15,8 @@ fn main() {
     let ffigen = FfiGen::new(&path).unwrap();
     let dart = dir.join("dart").join("lib").join("tlfs.dart");
     ffigen.generate_dart(dart, "tlfs", "tlfs").unwrap();
-    let js = dir.join("js").join("bindings.js");
+    let js = dir.join("js").join("bindings.mjs");
```

Comment:
whats mjs?

Reply:
So the thing should be imported as an ES module.